### PR TITLE
Change term request issue to GitHub form

### PR DESCRIPTION
**Pull request by jfy133**
New PR due to a mangled branch on my fork.

Please see the original discussion [here](https://github.com/GenomicsStandardsConsortium/mixs/pull/558).

### CODE REVIEW

**Comment by jfy133** [comment]
@turbomam you gave an original approval on this, could you read add for until we get the final OK for consistency

File: .github/ISSUE_TEMPLATE/term-request.md [hunk 1]
```diff
@@ -1,25 +1,88 @@
...
+        The pattern of the value that the term will be checked against. Previously known as 'Value syntax'.
+      placeholder: >
+        "e.g. {float} {unit}|{termLabel} {[termID]}|{text}|{timestamp} etc...".
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: Example
+      description: >
+        An example value for the term/slot
```
Comment (only1chunts) [inline .github/ISSUE_TEMPLATE/term-request.md:57]: should we extend this description to include "the example value must follow the pattern/syntax described above."

**Review by turbomam — APPROVED** [review]
I approve, pending a resolution to @only1chunts suggestion about  "the example value must follow the pattern/syntax described above."

File: .github/ISSUE_TEMPLATE/term-request.md (hunk 1 above)
Comment (jfy133) [inline .github/ISSUE_TEMPLATE/term-request.md:57]:
```suggestion
        An example value for the term/slot. The example value must follow the pattern/syntax described above.
```

**Comment by jfy133** [comment]
@only1chunts done! Please give ✅ of you're happy and I can merge

**Review by only1chunts — APPROVED** [review]

**Pull request merged**